lib/util/request.ts

```diff
@@ -2,11 +2,15 @@ import * as requestPromise from 'request-promise-native';
 import { Config, Request, TaxjarError } from '../util/types';
 
 const proxyError = (result): never => {
-  throw new TaxjarError(
-    result.error.error,
-    result.error.detail,
-    result.statusCode,
-  );
+  const isTaxjarError = result.statusCode && result.error && result.error.error && result.error.detail;
+  if (isTaxjarError) {
+    throw new TaxjarError(
+      result.error.error,
+      result.error.detail,
+      result.statusCode
+    );
+  }
+  throw result;
```

Comment:
We should add a [new line](https://eslint.org/docs/rules/padding-line-between-statements) before this `throw result;`.

Reply:
Don't think we actually have the `padding-line-between-statements` rule turned on, but updated anyway. Thanks! 👍